packages/flutter_tools/gradle/src/main/groovy/native_plugin_loader.groovy

```diff
@@ -0,0 +1,116 @@
+import groovy.json.JsonSlurper
```

Comment:
I believe this file should start with the copyright header:

```
// Copyright 2014 The Flutter Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

```

Reply:
Addressed in ea8b3a056d74003abc6d25785127d163298881c3